File: codility/CountFactors.js
function solution(N) {
    // Implement your solution here
    let counter = 0,
        d = 1;
    while (d <= N / d) {
        if (d === N / d) {
            counter++;
        } else if (N % d === 0) {
            counter += 2;
        }
        d++;
    }
    return counter;
}

File: codility/python/CountFactors.py
import math

def solution(N):
    # Implement your solution here
    count = 0
    sqrt = math.floor(math.sqrt(N))
    for i in range(1, sqrt+1):
        if N % i == 0:
            count+=1
    count *= 2
    if N == sqrt * sqrt:
        count -= 1
    return count